packages/angular_devkit/architect/src/architect.ts

```diff
@@ -318,16 +318,16 @@ export class Architect {
     // Create private jobs.
     privateArchitectJobRegistry.register(_getTargetOptionsFactory(_host));
     privateArchitectJobRegistry.register(_getBuilderNameForTargetFactory(_host));
-    privateArchitectJobRegistry.register(_validateOptionsFactory(_host, _registry));
+    privateArchitectJobRegistry.register(_validateOptionsFactory(_host, this._registry));
```

Comment:
`_registry` is a parameter of the constructor (this code block).  Using `this` is not required since it is also local.

Reply:
Umm... TypeScript is warning that registry is unused since `_registry` is marked with a `private` https://github.com/angular/angular-cli/blob/24d2776ec2067c42474f82614d29a64f91903ceb/packages/angular_devkit/architect/src/architect.ts#L314

Reply:
In that case, the `private` should be removed since it is not actually used as a class instance member.